3475/0/versioning_spec.rb

```diff
@@ -0,0 +1,30 @@
+# This method smells of :reek:UtilityFunction
+def version_ints(version)
+  version.split('.').map(&:to_i)
+end
+
+# This method smells of :reek:NilCheck
+# This method smells of :reek:UncommunicativeVariableName
+def at_least_ruby_version(version)
+  vints = version_ints(version)
+  ruby_vints = version_ints(RUBY_VERSION)
+  vints.zip(ruby_vints).all? { |v, rv| v.nil? || rv.nil? || v >= rv }
```

Comment:
`vints` variable name doesn't make any sense without context as well as `v` and `rv`.
Variable name should utilize fill word